File: app/java/src/main/java/com/example/production/domain/model/inventory/IssueInstructionDetail.java
package com.example.production.domain.model.inventory;

import lombok.*;

import java.math.BigDecimal;
import java.time.LocalDateTime;

/**
 * 払出指示明細データエンティティ
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class IssueInstructionDetail {
    private Long id;
    private String instructionNumber;
    private Integer lineNumber;
    private String itemCode;
    private Integer routingSequence;
    private BigDecimal issueQuantity;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: app/csharp/src/ProductionManagement.Domain/Models/Inventory/IssueInstructionDetail.cs
namespace ProductionManagement.Domain.Models.Inventory;

/// <summary>
/// 払出指示明細
/// </summary>
public class IssueInstructionDetail
{
    public long? Id { get; init; }

    public required string InstructionNumber { get; init; }

    public required int LineNumber { get; init; }

    public required string ItemCode { get; init; }

    public required int RoutingSequence { get; init; }

    public required decimal IssueQuantity { get; init; }

    public DateTime CreatedAt { get; init; }

    public DateTime UpdatedAt { get; init; }
}
